builtin/game/falling.lua

```diff
@@ -529,16 +529,14 @@ function core.check_single_for_falling(p)
 			if same and d_bottom.paramtype2 == "leveled" and
 					core.get_node_level(p_bottom) <
 					core.get_node_max_level(p_bottom) then
-				convert_to_falling_node(p, n)
-				return true
+				return convert_to_falling_node(p, n)
```

Comment:
This should only return a boolean and not `obj` (see `convert_to_falling_node`).

Reply:
Fixed